src/library/scala/collection/LinearSeqOptimized.scala

```diff
@@ -117,20 +117,20 @@ trait LinearSeqOptimized[+A, +Repr <: LinearSeqOptimized[A, Repr]] extends Linea
   }
 
   override /*TraversableLike*/
-  def foldLeft[B](z: B)(f: (B, A) => B): B = {
+  def foldLeft[B](z: B)(@deprecatedName('f) op: (B, A) => B): B = {
```

Comment:
+1 for `op`. I wish I knew why op feels more natural. Is it because we're assuming a binop?